src/yunohost/hook.py

```diff
@@ -288,10 +289,8 @@ def hook_callback(action, hooks=[], args=None, no_trace=False, chdir=None,
             else:
                 post_callback(name=name, priority=priority, path=path,
                               succeed=True)
-            try:
-                result[state][name].append(path)
```

Comment:
Are you sure there are no ways to get several path for a same name ? For example with custom hook ?

Reply:
I think I changed that because it was a request from @alexAubin . But not sure.

Can @alexAubin  confirm this ?